kie-wb/kie-wb-distribution-wars/src/main/assembly/component-kie-wb-tomcat-7-common.xml

```diff
@@ -96,6 +96,7 @@
 
             <!-- Errai -->
             <exclude>WEB-INF/lib/errai-jboss-as-support-*.jar</exclude>
+            <exclude>WEB-INF/lib/errai-cdi-server-*.jar</exclude>
```

Comment:
@psiroky TL;DR yes.

errai-cdi-server is the 4.x version of errai-weld-integration. We finally renamed the jar as it hasn't been weld-specific in a long time and works with all CDI implementations. However, since the Tomcat build is using an old version of CDI (1.0) and therefore the old errail-weld-integration (3.0.6.Final), we can't have both errai-weld-integration and errai-cdi-server (for CDI 1.1+) on the classpath.

errai-jboss-as-support provids VFS mappings for classpath scanning only needed in WF, JBoss, EAP.